omnibus/config/software/datadog-agent-integrations.rb

```diff
@@ -47,14 +47,15 @@
   'kubernetes',
   'ntp',                           # provided as a go check by the core agent
 ]
-blacklist_req = Array.new
+
+# package names of dependencies that won't be added to the Agent Python environment
+blacklist_packages = Array.new
 
 if suse?
-  blacklist.push('aerospike')  # Temporarily blacklist Aerospike until builder supports new dependency
-  blacklist_req.push(/^aerospike==/)  # Temporarily blacklist Aerospike until builder supports new dependency
+  blacklist_folders.push('aerospike')  # Temporarily blacklist Aerospike until builder supports new dependency
+  blacklist_packages.push(/^aerospike==/)  # Temporarily blacklist Aerospike until builder supports new dependency
 end
 
-core_constraints_file = 'core_constraints.txt'
```

Comment:
I removed this constraints check to speed up iterations and because change after change it shrank down to a couple of dependencies, and only `wheel` might be actually important to enforce (not sure). IMO we should review the whole constraints logic.